apps/Downgrade Chromium/uninstall

```diff
@@ -7,6 +7,10 @@ function error {
   exit 1
 }
 
+reduceapt() { #remove unwanted lines from apt output
```

Comment:
There is no need to declare `reduceapt` here, as `api` functions are available to the scripts declaring `reduceapt` in the `api` script makes more sense.